virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -66,6 +66,10 @@ This release adds new features and enhancements related to the following compone
 
 [id="virt-4-20-virtualization_{context}"]
 === Virtualization
+//CNV-61642
+**Monitor AAQ resource quota usage with new metrics**
+
+The Application Aware Quota (AAQ) controller now exposes two new Prometheus metrics: `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp`. You can use these metrics to gain better visibility into the current usage and creation times of application-aware resource quotas, helping you effectively monitor and manage your OpenShift Virtualization resources.
```

Comment:
```suggestion
* You can now use the `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp` metrics to query the current usage and creation times of the Application-Aware Quota (AAQ) Operator resources.
```

Link to the feature documenation from the release note after https://github.com/openshift/openshift-docs/pull/96517 is merged. As a side note, use the variable {VirtProductName} to refer to OpenShift Virtualization.